04/app.js

```diff
@@ -1,23 +1,39 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
+import React from "react";
+import { createRoot } from "react-dom/client";
 
-import File from './File';
-import List from './List';
+import File from "./File";
+import List from "./List";
 
 class App extends React.Component {
-    state = {
-        filesList: [],
-    }
+  state = {
+    filesList: [],
+  };
 
-    render() {
-        return (
-            <section>
-                <File />
-                <List />
-            </section>
-        )
-    }
+  filesListToState(array) {
```

Comment:
ooo tutaj właśnie przekazujesz dane przez parametr, a sama funkcje jest uruchamiana dopiero w `File` - w poprzednich zadaniach też o to chodziło.